backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -169,6 +178,10 @@ public void setFacility(Facility f) {
     facility = f;
   }
 
+  public void setPrimaryPhone(PhoneNumber phoneNumber) {
```

Comment:
this needs a getter right?

Reply:
also do we need to validate somewhere that primaryPhone is a member of phoneNumbers?

Reply:
It has one, but the primary phone number is returned as the backwards-compatible `telephone`:

https://github.com/CDCgov/prime-simplereport/blob/b449fde556b64f392ccbd272112cca837af90a17/backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java#L217-L222

That might answer your second question as well: the caller is responsible for doing that validation.